projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -262,6 +287,58 @@ public function update_item( $request ) {
 		return $this->create_item( $request );
 	}
 
+	/**
+	 * Permission check for updating last status.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return bool|WP_Error
+	 */
+	public function update_last_status_permissions_check( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		return $this->update_item_permissions_check( $request );
+	}
+
+	/**
+	 * Updates last status of an existing update schedule.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response|WP_Error The updated event or a WP_Error if the schedule could not be found.
+	 */
+	public function update_last_status( $request ) {
+		$events   = wp_get_scheduled_events( 'jetpack_scheduled_update' );
+		$last_run = get_site_transient( 'jetpack_scheduled_update_last_run' );
+
+		if ( false === $last_run || ! isset( $events[ $last_run ] ) ) {
+			return new WP_Error( 'rest_invalid_schedule', __( 'The last schedule could not be found.', 'jetpack-scheduled-updates' ), array( 'status' => 404 ) );
+		}
+
+		$last_statuses = get_option( 'jetpack_scheduled_update_last_statuses', array() );
+		$option        = array();
+
+		if ( ! is_array( $last_statuses ) ) {
+			$last_statuses = array();
+		}
```

Comment:
I don't think that's necessary. It's a new option, we know it's an array—let's instill confidence in future code readers that this will always be an array.